joomla/joomla-cms
# End of support plugin

Pull Request for Issue # .

### Summary of Changes
Rewriting the 3.10 End of Service Notification and placing it into Joomla 4 coding structure and style. 
Simplified the language definitions to not be hardcoded for 3 to 4
Currently using const EOS_DATE = '2023-10-25'; in file plugins/quickicon/eos/src/Extension/Eos.php The correct date should be 2025-10-25 but then the notification does not appear. 
Dimitris converted the jQuery code into working javascript for closing the notification and also calling the com_ajax so the hide me is stored in the plugin parameters.


### Testing Instructions
Build. 
Check plg_quickicon_eos is enabled. 
View Home Dashboard in the back end.


### Actual result BEFORE applying this Pull Request
No notification is displayed


### Expected result AFTER applying this Pull Request
A notification is displayed


### Link to documentations
Please select:
- [x ] No documentation changes for docs.joomla.org needed
